sipservice/src/main/java/com/phone/sip/PhoneComService.kt

```diff
@@ -8,19 +8,20 @@ import com.phone.sip.models.ConfigureSip
 
 class PhoneComService(
     var context: Context? = null,
-    var configurePushNotification: ConfigureFCMPushNotification?,
     var configureSip: ConfigureSip?,
     var configureServiceNotification: ConfigurePhoneServiceNotification?,
-    var enableSipLogging: Boolean
+    var enableSipConsoleLogging: Boolean,
+    var enableSipFileLogging: Boolean,
+    var logFilePath: String
 ) {
 
     private constructor(builder: Builder) : this(
         builder.context,
-        builder.configurePushNotification,
         builder.configureSip,
         builder.configureServiceNotification,
-        builder.enableSipLogging
-
+        builder.enableSipConsoleLogging,
```

Comment:
Now we have multiple parameters for enable log related functionality, shall we create a kind of Configuration class for Logging as well..